qndiag/tests/test_qndiag.py

```diff
@@ -19,7 +19,8 @@ def test_qndiag(weights, ortho):
     C = np.array([A.dot(d[:, None] * A.T) for d in diagonals])  # dataset
     if weights:
         weights = rng.rand(n)
-    B, _ = qndiag(C, weights=weights)  # use the algorithm
+    B, _ = qndiag(C, weights=weights,
+                  verbose=True, B0=np.eye(p), ortho=ortho)  # use the algorithm
```

Comment:
why the verbose? BO is not eye(p) by default?

Reply:
Indeed the verbose is unncessary. No B0 yields directly the good solution otherwise so algorithms don't even have to run.